authlib/jose/rfc7516/jwe.py

```diff
@@ -139,6 +141,145 @@ def serialize_compact(self, protected, payload, key, sender_key=None):
             urlsafe_b64encode(tag)
         ])
 
+    def serialize_json(self, header_obj, payload, keys, sender_key=None):
+        if isinstance(keys, tuple):  # tuple instead of list
+            keys = list(keys)
+        if not isinstance(keys, list):  # single key
+            keys = [keys]
+
+        if not keys:
+            raise ValueError("No keys have been provided")
+
+        shared_header = JWESharedHeader.from_dict(header_obj)
+
+        recipients = header_obj.get('recipients')
+        if recipients is None:
+            recipients = [{} for _ in keys]
+        for i in range(len(recipients)):
```

Comment:
```suggestion
        for recipient in recipients:
```

Reply:
No, otherwise replacement of a recipient being None with an empty dict will take no effect.